src/StackExchange.Metrics/Handlers/StatsdMetricHandler.cs

```diff
@@ -13,10 +13,104 @@
 
 namespace StackExchange.Metrics.Handlers
 {
+    /// <summary>
+    /// Implements <see cref="IMetricHandler" /> by sending data to a statsd endpoint.
+    /// </summary>
+    public class StatsdMetricHandler : IMetricHandler
```

Comment:
You might want to consider adding the `MaxPayloadCount` property too (https://github.com/StackExchange/StackExchange.Metrics/blob/main/src/StackExchange.Metrics/Handlers/SignalFxMetricHandler.cs#L63-L80), so that the underlying buffered handler can be configured